docs/src/pages/components/flex/react-auto-prop-table-flex.mdx

```diff
@@ -0,0 +1,128 @@
+import { Expander, ExpanderItem } from '@aws-amplify/ui-react';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+## Props
+
+<Expander type="multiple" defaultValue={['Props']}>
+  
+<ExpanderItem title="Amplify UI Props" value="Props">
+    <ReactMarkdown
+      remarkPlugins={[remarkGfm]}
+      children={`
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| ariaLabel | string | / |  |
+| ariaValuetext | string | / |  |
+| as | ElementType | / |  |
+| className | string | / | Additional CSS class name for component |
+| id | string | / | Unique identifier |
+| isDisabled | boolean | / |  |
+| ref | Ref&#60;HTMLDivElement&#62; | / |  |
+| role | React.AriaRole | / |  |
+| style | CSSProperties | / |  |
+| testId | string | / | Used to provide a &#96;data-testid&#96; attribute for testing purposes |
+
+      `}
+    />
+
+  </ExpanderItem>
+
+<ExpanderItem title="Layout" value="Layout">
+    <ReactMarkdown
+      remarkPlugins={[remarkGfm]}
+      children={`
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| alignContent | ResponsiveStyle&#60;AlignContent&#62; | / | Sets the distribution of space between and around content items |
```

Comment:
I don't think users are going to know what a ResponsiveStyle is, I think pretty much all the these can just be strings (need to verify).